### DISCUSSION TITLE
Year 2000 compset NF2000climo

### DISCUSSION THREAD

**Discussion by oyvindseland** [NorESMhub/CAM#79] [General]
NF2000climo is a useful compset for development but since it was not used for CMIP6 the settings have not been updated. I have started this issue to report on improving this compset

**Comment by oyvindseland** [comment]
Does not work for the CMIP6 release neither in LM nor MM version

**Comment by gold2718** [comment]
We should revisit this once we have settled on a physics suite for the next release of NorESM.
I could try to resurrect a working CAM6.3-based compset, however, it would not match CMIP6 simulations.

**Comment by oyvindseland** [comment]
Works now in NorESM3 development version
